packetery/src/Packetery/Module/Order/LabelPrint.php

```diff
@@ -161,8 +208,44 @@ public function hideFromMenus(): void {
 	 * Prepares labels.
 	 *
 	 * @param int $offset Offset value.
+	 *
+	 * @return Response\PacketsLabelsPdf|null
+	 */
+	private function prepareLabels( int $offset ): ?Response\PacketsLabelsPdf {
+		$orderIds        = explode( ',', $this->httpRequest->getQuery( 'orderIds' ) );
```

Comment:
nestane se, že při tisku většího množství zásilek nebude stačit max délka url ?  Pro eshopy není problém tisknou 500 štítků najednou